api/src/opentrons/protocol_engine/execution/command_executor.py

```diff
@@ -138,15 +180,27 @@ async def execute(self, command_id: str) -> None:
                 )
             )
         else:
-            completed_command = running_command.copy(
-                update={
-                    "result": result,
-                    "status": CommandStatus.SUCCEEDED,
-                    "completedAt": self._model_utils.get_timestamp(),
-                }
+            update = {
+                "result": result,
+                "status": CommandStatus.SUCCEEDED,
+                "completedAt": self._model_utils.get_timestamp(),
+            }
+            update.update(
+                _append_notes_if_notes(running_command, note_tracker.get_notes())
             )
+            completed_command = running_command.copy(update=update)
```

Comment:
No preference from me, but you could write this like this if you wanted to avoid the `update.update` thing.

```suggestion
            update = {
                "result": result,
                "status": CommandStatus.SUCCEEDED,
                "completedAt": self._model_utils.get_timestamp(),
                **_append_notes_if_notes(running_command, note_tracker.get_notes())
            }
            completed_command = running_command.copy(update=update)
```